gnat2goto/driver/driver.adb

```diff
@@ -164,6 +191,75 @@ package body Driver is
          Append_Op (Start_Body, Initialization_Statement);
       end Initialize_CProver_Rounding_Mode;
 
+      procedure Initialize_CProver_Dead_Object;
```

Comment:
So these three procedure only differ in the `Identifier`?

Reply:
pretty much. I can abstract if if you prefer?

Reply:
I'm not insisting.